blueprints/controllers/ikea_e1812/ikea_e1812.yaml

```
@@ -147,7 +147,7 @@ variables:
       button_release: ['1003']
     zha:
       button_short: ['on']
-      button_long: [move_with_on_off_0_83]
+      button_up_long: [move_with_on_off_0_83, 'move_with_on_off_MoveMode.Up_83']
```

```suggestion
      button_long: [move_with_on_off_0_83, 'move_with_on_off_MoveMode.Up_83']
```
I think the correct action name should be `button_long` here, otherwise it wouldn't be possible to select the correct action as there's no "up" button - just a single one. :)